kzg/src/eip_7594.rs

```diff
@@ -153,20 +156,45 @@ fn compute_fk20_proofs<
         }
     }
 
-    for i in 0..k2 {
-        h_ext_fft[i] = TG1::g1_lincomb(
-            s.get_x_ext_fft_column(i),
-            &coeffs[i],
-            FIELD_ELEMENTS_PER_CELL,
-            None,
-        );
+    #[cfg(feature = "parallel")]
+    {
+        h_ext_fft.par_iter_mut().enumerate().for_each(|(i, h_ext_fft_elem)| {
+            *h_ext_fft_elem = TG1::g1_lincomb(
+                s.get_x_ext_fft_column(i),
+                &coeffs[i],
+                FIELD_ELEMENTS_PER_CELL,
+                None,
+            );
```

Comment:
g1_lincomb is already parallelized, but due to small MSM size this doesn't give any performance penalty, but still not a good thing. Should probably be removed for now, later we'll add g1_lincomb_batch trait, that will run `N` parallel small MSMs, or something like that.

Reply:
Got it, parallelization is removed.